app/index.html

```diff
@@ -43,8 +43,12 @@
     <body>
 
         <header class="header" ng-include="'partials/header.html'"></header>
-        <div class="sidebar" ng-include="'partials/sidebar.html'" ng-class="{ 'open': sidebarOpen }"></div>
-        <ng-view></ng-view>
+        <div class="sidebar" ng-scrollable>
+            <div ng-include="'partials/sidebar.html'" ng-class="{ 'open': sidebarOpen }"></div>
+        </div>
+        <div class="main-view" ng-scrollable>
+            <ng-view></ng-view>
+        </div>
```

Comment:
okay makes sense then, then you just need to fix the styling on mobile and tablet